homework_1/fizzbuzz/fizzbuzz_test/fizzbuzz.go

```diff
@@ -0,0 +1,57 @@
+package fizzbuzz
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"strconv"
+	"strings"
+)
+
+
+
+func Fizzbuzzgame(start int , end int) ([]string, error){
+
+	if start < 0 {
+		return nil, errors.New("Start is bigger then end")
+
+	}
+	if end < 0 {
+		return nil, errors.New("End is less then 0")
+
+	}
+	if end < start {
+		return nil, errors.New("End is lesser then start")
+	}
+	var fizbuzzlist []string
```

Comment:
It would probably be good to place an empty line between these two lines as you're logically doing different things (in the few lines above you were parsing command line arguments, while now you are starting to generate the output).